docs/cloudformation/example.template.yaml

```diff
@@ -0,0 +1,287 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: Janus - AWS login federation
+
+# !! This template is provided as an example and is not intended for production use !!
+--!! This line invalidates the template and will need to be removed before CloudFormation !!--
+
+Metadata:
+  AWS::CloudFormation::Interface:
+    ParameterGroups:
+      - Label:
+          default: S3 Configuration
+        Parameters:
+          - DistBucketName
+      - Label:
+          default: Security
+        Parameters:
+          - AccessRestrictionCidr
+          - SSHAccessCidr
+          - KeyName
+      - Label:
+          default: Networking
+        Parameters:
+          - VpcId
+          - Subnets
+          - TLSCert
+
+Parameters:
+  Subnets:
+    Description: The subnets where Janus will run
+    Type: List<AWS::EC2::Subnet::Id>
+  VpcId:
+    Description: The VPC in which Janus will run
+    Type: AWS::EC2::VPC::Id
+  KeyName:
+    Description: Name of the SSH keypair to put on the instance
+    Type: AWS::EC2::KeyPair::KeyName
+  Stage:
+    Description: Environment name
+    Type: String
+    AllowedValues:
+      - PROD
+  AMI:
+    Description: Base AMI for Janus instances
```

Comment:
It occurs to me that without the userdata, this may not be the case. The only thing is that the build.sbt already includes a bunch of code for producing an installable `deb` file for running Janus (which assumes a Debian derivative like Ubuntu).